frontend/controller/actions/identity.js

```diff
@@ -92,7 +90,7 @@ export default sbp('sbp/selectors/register', {
       return userId
     } catch (e) {
       console.error('gi.actions/identity/login failed!', e)
-      throw new GIErrorUIRuntimeError(L('Failed to login: {codeError}', { codeError: e.message }))
+      throw new GIErrorUIRuntimeError(L('Failed to login: {reportError}', LError(e)))
```

Comment:
@taoeffect here we have an interesting scenario: when this error happens, it doesn't make sense to ask the user to report us the error because it's a controlled / predictable / custom error.

![image](https://user-images.githubusercontent.com/14869087/78981339-a1f6ed00-7b17-11ea-9313-087994d97574.png)

So, my idea to overcome this problem is to throw a custom GIError instead of the default `TypeError`, so that `LError()` can detect it and not show the link. (See `translations.js`).

I remember a while ago you said we should not use `GIErrorUIRuntimeError` in this file but I don't remember why and now I even thing it's beneficial. Even if there's a problem, we can create another custom error. (a suggestion is welcome).

Anyway, with this solution, now the error is displayed correctly:

![image](https://user-images.githubusercontent.com/14869087/78981859-c69f9480-7b18-11ea-83ff-e9f3eef7f37c.png)

Reply:
For some reason your comment is appearing in `actions/identity.js`, so it doesn't make much sense.  I think you might have meant to write it under `contracts/group.js`?

> I remember a while ago you said we should not use GIErrorUIRuntimeError in this file

Not in `contracts/group.js`, but yes in `actions/identity.js`.

I don't see why the change is needed from TypeError in group.js

>  when this error happens, it doesn't make sense to ask the user to report us the error because it's a controlled / predictable / custom error.

Agreed with that, but again I'm confused because the lines about have nothing to do with changing the mincome but with logging in and signing up...?